app/assets/stylesheets/styles.css.scss

```diff
@@ -169,6 +169,37 @@ nav {
   p {
     font-size: 1.6rem;
     padding: 0 2rem;
+    padding-bottom: 2rem;
+  }
+}
+
+.tips {
+  .row {
+    margin: 0 50px;
+  }
+
+  p {
+    font-size: 1.6rem;
+    padding: 0 2rem;
+  }
+
+  ul {
+    font-size: 1.6rem;
+    padding: 0 6rem;
+  }
+
+  .sectionHead {
```

Comment:
I don't know if we have a class name style but tips-info below is not camel like this